appveyor.yml

```diff
@@ -1,81 +1,62 @@
+
+os: unstable
+
+# Fix line endings on Windows
+init:
+  - git config --global core.autocrlf true
+
 # environment variables
 environment:
   node_pre_gyp_accessKeyId:
     secure: aSOwv7didRWblartyZgpyr622Ey1/3SgYFhp0r3YDp0=
   node_pre_gyp_secretAccessKey:
     secure: od0SlCZXrCJXKTTgs3EXAZOBRxJShMd9wAW7Fya6I98JYgrp+tKBzKTqKgsZsLvq
 
-
+  matrix:
+  - nodejs_version: "0.10"
+  - nodejs_version: "0.12"
+  - nodejs_version: "4.4"
+  - nodejs_version: "5.11"
```

Comment:
I think we *do* want it. However, I think there might be a few breaking changes in there. In any case, a separate PR does not seem like a bad idea, no?